src/tvshow/keywords.rs

```diff
@@ -0,0 +1,155 @@
+//! https://developer.themoviedb.org/reference/tv-series-keywords
+
+use std::borrow::Cow;
+
+use crate::common::keyword::Keyword;
+
+/// Command to get the keywords of a tvshow.
+///
+/// ```rust
+/// use tmdb_api::prelude::Command;
+/// use tmdb_api::Client;
+/// use tmdb_api::client::reqwest::ReqwestExecutor;
+/// use tmdb_api::tvshow::keywords::TVShowKeywords;
+///
+/// #[tokio::main]
+/// async fn main() {
+///     let client = Client::<ReqwestExecutor>::new("this-is-my-secret-token".into());
+///     let cmd = TVShowKeywords::new(1);
+///     let result = cmd.execute(&client).await;
+///     match result {
+///         Ok(res) => println!("found: {:#?}", res),
+///         Err(err) => eprintln!("error: {:?}", err),
+///     };
+/// }
+/// ```
+#[derive(Clone, Debug, Default)]
+pub struct TVShowKeywords {
+    pub tv_show_id: u64,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct TVShowKeywordsResult {
+    pub id: u64,
+    pub results: Vec<Keyword>,
+}
+
+impl TVShowKeywords {
+    pub fn new(tv_show_id: u64) -> Self {
+        Self { tv_show_id }
+    }
+}
+
+impl crate::prelude::Command for TVShowKeywords {
+    type Output = TVShowKeywordsResult;
```

Comment:
I'm wondering what the `id` field in this struct refers to. If it's only the TV show id, then we could just consider `Vec<Keyword>` as the output type.